acceptance/builder_test.go

```diff
@@ -0,0 +1,412 @@
+package acceptance
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle/api"
+	"github.com/buildpacks/lifecycle/platform"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+var (
+	builderDockerContext = filepath.Join("testdata", "builder")
+	builderBinaryDir     = filepath.Join("testdata", "builder", "container", "cnb", "lifecycle")
+	builderImage         = "lifecycle/acceptance/builder"
+)
+
+func TestBuilder(t *testing.T) {
+	//FIXME: test on windows and delete this skip condition if everything is fine
+	h.SkipIf(t, runtime.GOOS == "windows", "builder acceptance tests are not yet supported on Windows")
+
+	rand.Seed(time.Now().UTC().UnixNano())
+	//h.MakeAndCopyLifecycle(t, "linux", "amd64", builderBinaryDir)
```

Comment:
Having this line commented out is causing the linter to fail because `builderBinaryDir` is now unused. I think we could bring it back (and line 40).